food-and-agriculture-organization-of-the-united-nations.csl

```diff
@@ -10,196 +10,368 @@
-              <text macro="editor"/>
-              <text macro="publisher"/>
-            </if>
-            <else>
-              <text variable="title" font-style="italic" suffix="."/>
-              <text macro="editor"/>
-              <text macro="publisher"/>
-            </else>
-          </choose>
-        </if>
-        <else-if type="chapter" match="any">
-          <text variable="title" prefix=" " suffix=". "/>
-          <text term="in" text-case="capitalize-first" font-style="italic"/>
-          <group delimiter=" " prefix=" ">
-            <text macro="editor"/>
-            <text variable="container-title" font-style="italic" suffix=", "/>
-            <text variable="collection-title" prefix=" " suffix="."/>
-          </group>
-          <group suffix=".">
-            <label variable="page" suffix="." form="short"/>
-            <text variable="page"/>
-          </group>
-          <group suffix=".">
-            <text macro="publisher" prefix=" "/>
-            <group prefix=", ">
-              <text variable="number-of-pages" suffix=" "/>
-              <label variable="page" form="short"/>
-            </group>
-          </group>
-        </else-if>
-        <else-if type="webpage" match="any">
-          <text variable="title" prefix=" " suffix=". "/>
-          <text macro="Available-at-and-accessed"/>
-        </else-if>
-        <else-if type="article-newspaper article-magazine" match="any">
-          <text variable="title" suffix=". "/>
-          <text variable="container-title" form="short" font-style="italic" suffix="."/>
-          <group prefix=" ">
-            <choose>
-              <if match="any" variable="URL">
-                <text macro="Available-at-and-accessed" prefix=" "/>
-              </if>
-            </choose>
-          </group>
-        </else-if>
-        <else-if type="report" match="any">
-          <text variable="title" font-style="italic" suffix="."/>
-          <text macro="publisher"/>
-          <group prefix=" ">
+        <if match="none" type="legal_case">
+          <!-- Every item type except legal case uses "Anonymous" as author if there's no author. So legal case is singled out here. -->
+          <group delimiter=". ">
+            <text macro="author" font-weight="bold"/>
+            <text macro="date"/>
             <choose>
-              <if match="any" variable="URL">
-                <text macro="Available-at-and-accessed"/>
+              <if type="book">
+                <!-- book and computer program in Zotero -->
+                <group>
+                  <text variable="title" font-style="italic"/>
+                  <text variable="medium" prefix=" [" suffix="]"/>
+                  <!-- For computer program in Zotero. Can put "CD&#8211;ROM / DVD&#8211;ROM" in "System" field. -->
+                  <text variable="genre" prefix=" [" suffix="]"/>
+                  <!-- For eBook  in Mendeley. Can put "Kindle edition" in "Genre" field. -->
```

Comment:
btw. you can do that in Zotero, too, by putting `genre: Kindle edition` into extra.